src/app/modules/attribution/start-maprule.html

```diff
@@ -1,25 +1,17 @@
-
 <div class="col-md-12">
 	<view-maprule></view-maprule>
 </div>
 <div class="col-md-12">
-	<div class="content-group start-group container">
-		<div class="row">
-			<div class="col-md-2">
-				<button class="btn btn-basic start-button" (click)="toEdit()">
-					Edit MapRule
-				</button>
-			</div>
-			<div class="col-md-2">
-				<div ngbDropdown>
-					<button class="btn btn-success start-button" id="dropdownBasic1" ngbDropdownToggle>
-					Map With
-					</button>
-					<div ngbDropdownMenu aria-labelledby="dropdownBasic1" class="start-button">
-						<a id="iDlink" class="dropdown-item" target="_blank">iD editor</a>
-						<a id="josmLink" class="dropdown-item" target="_blank">JOSM</a>
-					</div>
-				</div>
+	<div class="content-group start-group-container d-flex justify-content-center">
+		<button class="btn btn-lg start-button" id="edit-button" (click)="toEdit()">Edit MapRule</button>
+		<ng-template #tipContent>
+			<div class="d-flex flex-column">
+				<a href="{{ idUrl }}" id="iDlink" class="dropdown-item edit-with-item" target="_blank">iD editor</a>
+				<a href="{{ josmUrl }}" id="josmLink" class="dropdown-item edit-with-item" target="_blank">JOSM</a>
 			</div>
+		</ng-template>
+		<button type="button" class="btn btn-lg start-button" triggers="click" [ngbTooltip]="tipContent" tooltipClass="edit-with-tooltip">
+			Map With
```

Comment:
Can we add an arrow or add some indication like "..." here to show that it will open opentions
![map with](https://user-images.githubusercontent.com/34665975/47118716-32f84b00-d236-11e8-8c97-7d898ab1cbf2.png)